docs/docs/sourcing-from-netlify-cms.md

```diff
@@ -77,11 +77,11 @@ collections:
       - { name: body, label: Body, widget: markdown }
 ```
 
-Then in your terminal run `npm run develop` to start the Gatsby development server. Once the server
-is running, it will print the address to open for viewing. It's typically `localhost:8000`. Open that
-in a browser and you should see the text "Hello World" in the top left corner. Now navigate to
-`/admin/` - so if your site is at `localhost:8000`, go to `localhost:8000/admin/`. **The trailing
-slash is required!**
+Then in your terminal run `npm run build` to build the site then `npm run develop` to start the
```

Comment:
```suggestion
Then in your terminal run `gatsby develop` to start the
```

Reply:
The PR is trying to add documentation for working with Netlify CMS locally(vs a remote CMS with local Gatsby development if I understand correctly. However the author has mistakenly mixed up build/serve calls as build/develop (which afaik invalidates the build output?).

It should be left as it was previously, with a comment linking to the docs the author provided here, or otherwise mention the build/serve workflow when using Netlify CMS locally? (I might have misunderstood the linked docs, I don't use Netlify CMS, but the instructions clearly make a distinction between develop and build/serve).